charts/openmetadata/templates/deployment.yaml

```diff
@@ -71,6 +71,7 @@ spec:
             value: "{{ .Values.global.openmetadata.port }}"
           - name: SERVER_ADMIN_PORT
             value: "{{ .Values.global.openmetadata.adminPort }}"
+            {{- include "check_provider_value" . }}
```

Comment:
This is not required!

Reply:
Let's remove this code!

Reply:
Let's remove `{{- include "check_provider_value" . }}` this piece of code!

Reply:
removed the code {{- include "check_provider_value" . }}